H2MLauncher.UI/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "H2MLauncher": {
-    "MWRLocation": "weweewew"
```

Comment:
Why did you remove MWRLocation?

Reply:
The default should be empty which is the current directory (relative path). When the game is first detected i set it to the full path if its empty in #58 (https://github.com/Bowhza/H2M-Launcher/pull/58/files#diff-a65fb504b4ad5b079ce84653e7c83bdb1feff480d9b85b6149972d4e0a5667edR87)